src/resolvers/Mutation/createVenue.ts

```diff
@@ -111,7 +111,7 @@ export const createVenue: MutationResolvers["createVenue"] = async (
   }
 
   // Check if the venue name provided is an empty string.
-  if (!args.data?.name ?? "") {
+  if (!args.data?.name) {
```

Comment:
In TypeScript 5.0 and above, there is a stricter check on the usage of the nullish operator `(??)` and the optional chaining operator `(?.)`. Specifically, TypeScript now issues an error when the left operand of `??` is an expression that can never be null or undefined, making the right operand unreachable.

The expression `!args.data?.name` results in a `boolean`
Using `?? "" `after a boolean is unnecessary because a boolean is never null or undefined. The nullish operator `?? `is intended to handle null or undefined values, not booleans.
Therefore, TypeScript flags this as an error for the same reason as above in latest upgrade 

Ref: https://github.com/microsoft/TypeScript/pull/59217